src/Types/Scripts.purs

```diff
@@ -10,7 +10,17 @@ module Types.Scripts
 
 import Prelude
 
-import Data.Argonaut (class DecodeJson, class EncodeJson)
+import Contract.Prelude (Either(..))
```

Comment:
We should avoid importing `Contract` modules internally

Reply:
How did that get here, I guess IDE being very smoort again